Backend/DataAccessLayer/Repositories/UserManagementRepository.cs

```diff
@@ -67,5 +67,17 @@ public bool ExistingUser(DatabaseContext _db, string email)
             }
             return false;
         }
+
+        public bool IsManagerOf(DatabaseContext _db, User user, User subject)
+        {
+            if (subject == null || subject.ManagerId == null) return false;

```

Comment:
Perform the same check for the User user object

Reply:
This check is for recursion. The first parameter is never changed. Are you suggesting this to protect consumers of the repo down the road?